node/network/availability-distribution/src/responder.rs

```diff
@@ -64,6 +64,12 @@ pub async fn answer_request<Context>(
 where
 	Context: SubsystemContext,
 {
+	let mut span = jaeger::candidate_hash_span(&req.payload.candidate_hash, "answer_request");
+	span.add_stage(jaeger::Stage::AvailabilityDistribution);
+	let _child_span = span.child_builder("answer_chunk_request")
```

Comment:
we're using kebab-case elsewhere, right?
```suggestion
	let mut span = jaeger::candidate_hash_span(&req.payload.candidate_hash, "answer-request");
	span.add_stage(jaeger::Stage::AvailabilityDistribution);
	let _child_span = span.child_builder("answer-chunk-request")
```